web/app/services/session.ts

```diff
@@ -112,7 +112,7 @@ export default class SessionService extends EmberSimpleAuthSessionService {
       }
     }
 
-    if (redirectTarget) {
+    if (redirectTarget && redirectTarget !== "/authenticate") {
```

Comment:
`redirectTarget` becomes `authenticate` when refreshing the login screen. In this case, we ignore it and fall back to the `routeAfterAuthentication` value of `'index'`.